app/test/test.js

```diff
@@ -10,81 +10,95 @@ const inkyPathsByPlatform = {
   "win32": "../Inky-win32-x64/Inky.exe"
 };
 
+const app = new Application({
```

Comment:
Pulling this out seems like it might introduce the possibility of tests not being independent; if app holds _any_ state, then it should be initialized before each test rather than once globally.